### PR TITLE
Update services

### PR DESCRIPTION
**Pull request by teo-jun-xiong**
* Updated services to include `/` at the end
* Set minimum width for `ProfileCollection`
* Add no images for this collection
* Link ProfileCollection to `collection/:collectionId`

### CODE REVIEW

File: src/components/button/LogoutButton.jsx [hunk 1]
```diff
@@ -11,8 +11,10 @@ const LogoutButton = () => {
   return (
     <IonButton
       onClick={() => {
-        localStorage.removeItem("access_token");
-        localStorage.removeItem("refresh_token");
+        localStorage.removeItem("accessToken");
+        localStorage.removeItem("refreshToken");
+        localStorage.removeItem("userId");
+        localStorage.removeItem("isGuest");
```
Comment (marcus2k) [inline src/components/button/LogoutButton.jsx:17]: For guest users, I think we can have a different Toolbar, probably only need Home, Discover, and Profile (to sign up or sign in)

File: src/components/form/CollectionForm.jsx [hunk 2]
```diff
@@ -12,25 +12,26 @@ const getDefaultCollectionData = () => {
...
 
-  console.log(categoryOptions);
+  useEffect(() => {
+    if (props.collectionData) {
+      setCollectionName(props.collectionData.collectionName);
+      setCollectionDescription(props.collectionData.collectionDescription);
+      setCategory(props.collectionData.categoryId);
+    }
+  }, [props.collectionData]);
+
```
Comment (marcus2k) [inline src/components/form/CollectionForm.jsx:28]: Can we check if this part is still needed i.e. if it works without this useEffect block?

File: src/pages/home/Home.jsx [hunk 3]
```diff
@@ -2,16 +2,18 @@ import { useEffect, useState } from "react";
...
+import { getCategories } from "../../services/categories";
 
 const Home = () => {
   const [name, setName] = useState("");
 
   useEffect(() => {
-    getCollections();
+    getCollections().then((res) => console.log(res));
+    // getCategories().then((res) => console.log(res));
   }, []);
```
Comment (marcus2k) [inline src/pages/home/Home.jsx:17]: Hmm not sure what this useEffect is doing..

File: src/pages/login/Login.jsx [hunk 4]
```diff
@@ -40,6 +36,7 @@ const Login = () => {
 
   const handleGuestLogin = async () => {
     server.defaults.headers["Authorization"] = null;
+    localStorage.setItem("isGuest", "isGuest");
```
Comment (marcus2k) [inline src/pages/login/Login.jsx:39]: Is it possible for the value of "isGuest" be a boolean instead?

File: src/pages/home/Home.jsx (hunk 3 above)
Comment (marcus2k) [reply src/pages/home/Home.jsx:17]: What I meant was, the result is not being used right?

File: src/components/form/CollectionForm.jsx (hunk 2 above)
Comment (teo-jun-xiong) [reply src/components/form/CollectionForm.jsx:28]: It will but there may be unnecessary executions

**Pull request merged**